testproject/Assets/Tests/Runtime/MultiprocessRuntime/Helpers/BokkenMachine.cs

```diff
@@ -0,0 +1,263 @@
+            MultiprocessLogger.Log($"\"dotnet {PathToDll} {command}\"");
+
+            var workerProcess = new Process();
+
+            workerProcess.StartInfo.FileName = Path.Combine("dotnet");
+            workerProcess.StartInfo.UseShellExecute = false;
+            workerProcess.StartInfo.RedirectStandardError = true;
+            workerProcess.StartInfo.RedirectStandardOutput = true;
+            workerProcess.StartInfo.Arguments = $"{PathToDll} {command} ";
+            try
+            {
+                MultiprocessLogger.Log($"Starting process : waitForResult is {waitForResult}");
+                var newProcessStarted = workerProcess.Start();
+
+                if (!newProcessStarted)
+                {
+                    throw new Exception("Failed to start worker process!");
+                }
+                else
+                {
+                    MultiprocessLogger.Log($"Started process : waitForResult is {waitForResult}");
+                }
+            }
+            catch (Win32Exception e)
+            {
+                MultiprocessLogger.LogError($"Error starting bokken process, {e.Message} {e.Data} {e.ErrorCode}");
+                throw;
+            }
+
+            if (waitForResult)
+            {
+                workerProcess.WaitForExit(timeToWait);
+                if (logStdOut)
+                {
+                    string so = workerProcess.StandardOutput.ReadToEnd();
+                    MultiprocessLogger.Log(so);
+                }
+                // string se = workerProcess.StandardError.ReadToEnd();
+                // MultiprocessLogger.LogError(se);
+            }
+            else
+            {
+                s_ProcessList.Add(workerProcess);
+            }
+            MultiprocessLogger.Log($"Execute Command End");
+            return workerProcess;
+        }
+
+        private string GenerateCreateCommand()
+        {
+            if (string.IsNullOrEmpty(PathToJson))
+            {
+                throw new Exception("PathToJson must not be null or empty");
+            }
+
+            string s = $" --command create --output-path {PathToJson} --type {Type} --image {Image} --flavor {Flavor} --name {Name}";
+            return s;
+        }
+
+        private string GenerateSetupMachineCommand()
+        {
+            if (string.IsNullOrEmpty(PathToJson))
+            {
+                throw new Exception("PathToJson must not be null or empty");
+            }
+
+            string s = $" --command setupmachine --input-path {PathToJson}";
+            return s;
+        }
+
+        private string GenerateLaunchCommand(string ip)
+        {
+            if (string.IsNullOrEmpty(PathToJson))
+            {
+                throw new Exception("PathToJson must not be null or empty");
+            }
+
+            if (Image.Contains("win10"))
+            {
+                LogPath = @"C:\users\bokken\.multiprocess\" + $"logfile-mp-{DateTimeOffset.Now.ToUnixTimeSeconds()}.log";
```

Comment:
I'm a yamato/bokken noob, but wouldn't those root paths be given by yamato/bokken themselves (like env variables, something like that?)

Reply:
I added a comment in the code that we are waiting for better support from the bokken API to make the use of environment variables easier but at the moment these magic hard-coded paths are needed.